include/ps/env.h

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2016 by Contributors
+ */
+#ifndef PS_ENV_H_
```

Comment:
please consider move env to internal/env.h since it is only used by internal implementation